synctoolui/src/main/java/org/duracloud/syncui/SyncUIDriver.java

```diff
@@ -85,16 +88,34 @@ private static void launchServer(final String url,
             contextPath = SyncUIConfig.getContextPath();
             Server srv = new Server(port);
 
+            // Setup JMX
+            MBeanContainer mbContainer = new MBeanContainer(

```

Comment:
@nwoodward :  would you mind just adding a comment with a link to the sample code you based this fix on?